packages/hint/docs/user-guide/development-flow-integration/index.md

```diff
@@ -0,0 +1,22 @@
+# Development flow integration
+
+`webhint` can be used at different steps of the development flow. The
```

Comment:
> during different ?

Reply:
That's more stylistic than anything else, right?